sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2489,7 +2493,9 @@ private static void ValidateActions(TextAnalyticsActions actions)
                 actions.ExtractKeyPhrasesActions?.Count > 1 ||
                 actions.AnalyzeSentimentActions?.Count > 1 ||
                 actions.ExtractSummaryActions?.Count > 1 ||
-                actions.SingleCategoryClassifyActions?.Count > 1)
+                actions.SingleCategoryClassifyActions?.Count > 1 ||
+                actions.MultiCategoryClassifyActions?.Count > 1
+                )
             {
                 throw new ArgumentException("Multiple of the same action is not currently supported.");
```

Comment:
probably not for this PR, but just wanted to check - the service team said multiple of the same action would be supported in preview.2. I've done quick tests to confirm that it works (at least with multiple RecognizePiiEntitiesActions). Do you have an issue open to allow for this in this beta?

Reply:
We have now. but yeah, def not for this PR

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/24430

Reply:
IIRC a related issue will be to expose `actionName` in the client libraries so inputs can be correlated with outputs.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/24441